tabular/src/autogluon/tabular/learner/abstract_learner.py

```diff
@@ -420,7 +420,7 @@ def leaderboard(self, X=None, y=None, extra_info=False, only_pareto_frontier=Fal
     # model: model (str) to get feature importances for, if None will choose best model.
     # features: list of feature names that feature importances are calculated for and returned, specify None to get all feature importances.
     # feature_stage: Whether to compute feature importance on raw original features ('original'), transformed features ('transformed') or on the features used by the particular model ('transformed_model').
-    def get_feature_importance(self, model=None, X=None, y=None, features: list = None, feature_stage='original', subsample_size=1000, silent=False) -> Series:
+    def get_feature_importance(self, model=None, X=None, y=None, features: list = None, feature_stage='original', subsample_size=1000, silent=False, **kwargs) -> DataFrame:
```

Comment:
1. That is getting into more complex logic for when we actually want to leverage feature importance in an AutoML setting. I think that can be discussed later on if it becomes an issue.
2. I think it makes sense to stratify based on target variable (added TODO), but not shuffled feature. The reasoning is that for feature importances to be comparable, they have to have the same shuffle order, and stratifying by feature would break that property.